internal/rbd/rbd_attach.go

```diff
@@ -235,6 +235,10 @@ func createPath(ctx context.Context, volOpt *rbdVolume, cr *util.Credentials) (s
 	if volOpt.readOnly {
 		mapOptions = append(mapOptions, "--read-only")
 	}
+
+	if volOpt.MapOptions != "" {
+		mapOptions = append(mapOptions, "--options", volOpt.MapOptions)
+	}
```

Comment:
If the intent is to supersede https://github.com/ceph/ceph-csi/pull/1450, a separate `UnmapOptions` is needed.

Reply:
@idryomov this option is to provide more user options, we still need to have defaulted in our code.

>a separate UnmapOptions is needed.

 I thought of adding it, do you have examples I can refer for supported unmap options?

Reply:
It's in the end of that same section, https://docs.ceph.com/docs/master/man/8/rbd/#kernel-rbd-krbd-options.  Currently it's just `force` (which shouldn't be used), but https://github.com/ceph/ceph/pull/36927 adds `noudev`.

Reply:
Thanks, @idryomov I have included unmap options also